applications/forms.py

```diff
@@ -56,24 +56,30 @@ class ApplicationForm(OverwriteOnlyModelFormMixin, BetterModelForm):
         label='I declare that I am at least 18 years of age.',
     )
 
-    extraLegalText = (
-        "I authorize you to share my application/registration information "
-        "for event administration, ranking, MLH administration, pre- and "
-        "post-event informational emails, and occasional messages about "
-        "hackathons in-line with the MLH Privacy Policy. Furthermore, I agree "
-        "to the terms of both the <a target='_blank' href='"
-        "https://github.com/MLH/mlh-policies/blob/master/"
-        "prize-terms-and-conditions/contest-terms.md'"
-        ">MLH Contest Terms and Conditions</a> "
-        "and the <a target='_blank' href='https://mlh.io/privacy'>"
-        "MLH Privacy Policy</a>."
-    )
-
     code_conduct = forms.BooleanField(required=False,
                                       label='I have read and agree to the '
-                                            '<a href="%s" target="_blank">MLH Code of Conduct</a>. %s' % (
-                                                getattr(settings, 'CODE_CONDUCT_LINK', '/code_conduct'),
-                                                extraLegalText), )
+                                            '<a href="%s" target="_blank">MLH Code of Conduct</a>.' %
+                                      getattr(settings, 'CODE_CONDUCT_LINK', '/code_conduct'), )
+
+    data_consent = forms.BooleanField(required=False,
+                                      label="I authorize you to share my application/registration information "
+                                      "for event administration, ranking, MLH administration, pre- and "
+                                      "post-event informational emails, and occasional messages about "
+                                      "hackathons in-line with the <a href='https://mlh.io/privacy' "
+                                      "target='_blank'>MLH Privacy Policy</a>.", )
+
+    sponsor_consent = forms.BooleanField(required=False,
+                                         label="I authorize you to share my resume, uploaded as part "
+                                         "of this application, with sponsors of the event.", )
```

Comment:
Mmm, I'd rephrase to "I would like my CV to be shared with the event sponsors"

Reply:
How about `I would like my CV, name, and year of graduation to be shared with the event sponsors`?

Reply:
Implemented.